content/applications/marketing/events/lead_generation_rules.rst

```diff
@@ -0,0 +1,206 @@
+=====================
+Lead generation rules
+=====================
+
+Odoo *Events* makes it possible for users to generate leads from their event attendees, based on a
+predetermined criteria, known as *Lead Generation Rules*. These rules can be customized in a number
+of different ways, in order to suit unique needs.
+
+Lead generation page
+====================
+
+In order to create lead generation rules for an event, users first must navigate to the *Lead
+Generation Rule* page inside of the Odoo *Events* application. To do so, navigate to
+:menuselection:`Events app --> Configuration --> Lead Generation`.
+
+On the :guilabel:`Lead Generation Rule` page, Odoo displays all the current lead generation rules
+that have been created for the database, in a standard list view.
+
+.. image:: lead_generation_rules/lead-generation-page.png
+   :align: center
+   :alt: The Lead Generation page in the Odoo Events application.
+
+From the :guilabel:`Lead Generation Rule` page, users can view pertinent information related to each
+existing lead generation rule, such as :guilabel:`Rule Name`, :guilabel:`When` the rule is
+configured to be triggered, any :guilabel:`Event Categories` it is associated with, the
+:guilabel:`Event` it is applied to, and the associating :guilabel:`Company` related to it (if
+applicable).
+
+Create lead generation rules
+============================
+
+To create a lead generation rule for an event, navigate to the :guilabel:`Lead Generation Rule` page
+in the Odoo *Events* application (:menuselection:`Events app --> Configuration --> Lead
+Generation`), then click :guilabel:`New`.
+
+Doing so reveals a blank :guilabel:`Lead Generation Rule` form.
+
+.. image:: lead_generation_rules/lead-generation-rule-form.png
+   :align: center
+   :alt: A standard Lead Generation Rule form in the Odoo Events application.
+
+First, start by entering a :guilabel:`Rule Name`.
+
+Then, designate a lead creation option for the lead generation rule in the :guilabel:`Create` field.
+The options are: :guilabel:`Per Attendee` or :guilabel:`Per Order`.
+
+If :guilabel:`Per Attendee` is selected, a lead is created for each attendee (B2C). If
+:guilabel:`Per Order` is selected, a single lead is created per ticket batch/sales order (B2B).
+
+Next, designate *when* the lead should be created in the :guilabel:`When` field. The options are:
+
+- :guilabel:`Attendees are created`: leads that follow the configurations of the rule are created
+  when an attendee record for an event is created.
+- :guilabel:`Attendees are registered`: leads that follow the configurations of the rule are created
+  when the attendee officially registers, either manually or automatically.
+- :guilabel:`Attendees attended`: leads that follow the configurations of the rule are created when
+  the attendee's attendance is officially confirmed, and the registration is set to *Done*.
```

Comment:
```suggestion
- :guilabel:`Attendees are created`: leads are created
  when an attendee record for an event is created.
- :guilabel:`Attendees are registered`: leads are created
  when the attendee officially registers, either manually or automatically.
- :guilabel:`Attendees attended`: leads are created when
  the attendee's attendance is officially confirmed, and the registration is set to *Done*.
```
Just trying to make these a bit less wordy, but if I changed the context too much, feel free to disregard